src/lib/utils/driplist/DripListService.ts

```diff
@@ -368,25 +346,13 @@ export default class DripListService {
     return ipfsHash;
   }
 
-  // We use the count of *all* NFT sub-accounts to generate the salt for the Drip List ID.
-  // This is because we want to avoid making HTTP requests to the subgraph for each NFT sub-account to check if it's a Drip List.
-  private _calcSaltFromAddress = (address: string, listCount: number): bigint /* 64bit */ => {
+  // Create random salt from address
```

Comment:
Are we sure this generates a _random_ number?
I think for the same address it will produce the same output, no?

Reply:
We're not sure and it looks like it doesn't based on the error I'm receiving. Will look into it!